File: lua/refactoring/tests/refactor/106/ts/with-comments/extract.expected.ts

function foo_bar(a: number, test, test_other) {
    for (let idx = test - 1; idx < test_other; ++idx) {
        console.log(idx, a)
    }
}


/*
    * This is a comment
    * wow!
*/
function simple_function(a: number) {
    let test = 1;
    let test_other = 11
    foo_bar(a, test, test_other);
}

File: lua/refactoring/tests/refactor/106/java/with-comments/extract.expected.java
class Extract {

public static void foo(int a, int test, int test_other) {
    for (int x = 0; x < test_other + test; x++) {
      System.out.println(x + " " + a);
    }
}


  /*
   * This is a comment
   * comments are fun
   * wow!
   */
  public static void simpleFunction(int a) {
    int test = 1, test_other = 11;
    foo(a, test, test_other);
  }
}
